### DISCUSSION TITLE
padlocal的微信版本问题

### DISCUSSION THREAD

**Discussion by chichihaixiaojian666** [wechaty/wechaty#2693] [Q&A]
padlocal grpc request failed: 1, error: Error: 2 UNKNOWN: [tid:a653f187] wechat bad request error: 你的应用版本过低，请升级至最新版本后再登录。点击“确定”后将跳转至最新版下载页面。

[a comment by an automated account is not shown]

**Reply by chichihaixiaojian666** [reply]
谢谢 请帮忙尽快解决

**Reply by chichihaixiaojian666** [reply]
"wechaty": "^1.20.2",
    "wechaty-puppet": "^1.20.2",
    "wechaty-puppet-padlocal": "^1.20.1"   版本都升级到最高了，还是一样子的。

**Reply by hedibin** [reply]
你好 你解决了吗

**Reply by Sakchai686868** [reply]
解决了吗？

[a comment by an automated account is not shown]

**Comment by Furowu** [comment]
I had the same reaction.
